code/datums/diseases/advance/symptoms/damage_converter.dm

```diff
@@ -42,6 +42,10 @@ Bonus
 
 		var/list/parts = H.get_damaged_organs(1,1) //1,1 because it needs inputs.
```

Comment:
So adding a third argument for flagging that kind of stuff isn't going to throw any errors for anything calling with only two args?